src/Controller/UI/ResearchGroupController.php

```diff
@@ -84,15 +87,15 @@ public function defaultAction(EntityHandler $entityHandler, int $id = null)
     /**
      * The Research Group Ladning page action.
      *
-     * @param integer       $id            The id of the entity to retrieve.
+     * @param string $id The id of the entity to retrieve.
      *
      * @throws NotFoundHttpException When the research group was not found.
      *
      * @Route("/research-group/about/{id}", name="pelagos_app_ui_researchgroup_about")
```

Comment:
Use requirment annotation instead, then type changes or not needed.
https://symfony.com/doc/4.4/routing.html#parameters-validation